src/platform/graphics/texture.js

```diff
@@ -37,7 +37,7 @@ class Texture {
     /** @protected */
     _isRenderTarget = false;
 
-    /** @protected */
+    /** @ignore */
     _gpuSize = 0;
```

Comment:
This can't be private as it is changed by both `WebglTexture` and `WebgpuTexture`.